code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -233,6 +233,9 @@
 	if(!istype(M))
 		return
 	if(isoozeling(M))
+		if(M.blood_volume <= 0)
+			M.blood_volume = 0
+			return
 		M.blood_volume -= 30
```

Comment:
The way this code is structured doesn't seem right to me. This means if they have 1 `blood_volume`, spraying them with water will reduce it to -14. If they get then sprayed again, *then* it will be set to 0. Also, if for any reason, valid or otherwise, their blood gets reduced below 0, spraying them with water will instantly set their blood to exactly 0.

Reply:
remind me to add a proc that removes blood and handles all this for us at some point, kube.

Reply:
About the H2O gas, that only applies on living, meaning it is not possible to go below 20% blood, as that is the minimum for a carbon to live. This gets called every time an oozeling contacts water, even if dead.